test/cmdlineTests/ir_optimized_transient_storage_value_type/output

```diff
@@ -0,0 +1,131 @@
+            }
+            function revert_error_42b3090547df1d2001c96683413b8cf91c1b902ef5e3cb8d9f6f304cf7446f74()
+            { revert(0, 0) }
+            function shift_right_0_unsigned(value) -> newValue
+            { newValue := shr(0, value) }
+            function cleanup_from_storage_uint256(value) -> cleaned
+            { cleaned := value }
+            function extract_from_storage_value_offsett_uint256(slot_value) -> value
+            {
+                value := cleanup_from_storage_uint256(shift_right_0_unsigned(slot_value))
+            }
+            function read_from_transient_storage_split_offset_uint256(slot) -> value
+            {
+                value := extract_from_storage_value_offsett_uint256(tload(slot))
+            }
+            function cleanup_rational_by(value) -> cleaned
+            { cleaned := value }
+            function cleanup_uint256(value) -> cleaned
+            { cleaned := value }
+            function identity(value) -> ret
+            { ret := value }
+            function convert_rational_by_to_uint256(value) -> converted
+            {
+                converted := cleanup_uint256(identity(cleanup_rational_by(value)))
+            }
+            function panic_error_0x11()
+            {
+                mstore(0, shl(224, 0x4e487b71))
+                mstore(4, 0x11)
+                revert(0, 0x24)
+            }
+            function checked_add_uint256(x, y) -> sum
+            {
+                x := cleanup_uint256(x)
+                y := cleanup_uint256(y)
+                sum := add(x, y)
+                if gt(x, sum) { panic_error_0x11() }
+            }
+            function shift_left(value) -> newValue
+            { newValue := shl(0, value) }
+            function update_byte_slice_shift(value, toInsert) -> result
+            {
+                let mask := not(0)
+                toInsert := shift_left(toInsert)
+                value := and(value, not(mask))
+                result := or(value, and(toInsert, mask))
+            }
+            function convert_uint256_to_uint256(value) -> converted
+            {
+                converted := cleanup_uint256(identity(cleanup_uint256(value)))
+            }
+            function prepare_store_uint256(value) -> ret
+            { ret := value }
+            function update_transient_storage_value_offsett_uint256_to_uint256(slot, value)
+            {
+                let convertedValue := convert_uint256_to_uint256(value)
+                tstore(slot, update_byte_slice_shift(tload(slot), prepare_store_uint256(convertedValue)))
+            }
+            function fun_f()
+            {
+                let _1 := read_from_transient_storage_split_offset_uint256(0x00)
+                let expr := _1
+                let expr_1 := 0x01
+                let expr_2 := checked_add_uint256(expr, convert_rational_by_to_uint256(expr_1))
+                update_transient_storage_value_offsett_uint256_to_uint256(0x00, expr_2)
```

Comment:
Hm, so that's extremely strange  because if you look at my changes in [`updateStorageValueFunction`](https://github.com/ethereum/solidity/pull/15292/files#diff-331d761d82437590a3abfd03a44ddd6b0abc88b54095707f69005276f5d1ac19R2915), you'll see that I've only introduced differentiation between transient storage and storage, and have not touched the offset portion of the name.

Also weird is that in the implementation, it's spelled as `offset`, where as in the test output, it's `offsett`. There's also an [older test](https://github.com/ethereum/solidity/blame/develop/test/cmdlineTests/inline_assembly_function_name_clash/output#L131) with this same issue.